src/FormatNegotiator.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Franzl\Middleware\Whoops;
+
+use Psr7Middlewares\Utils;
```

Comment:
Hi, it came from [oscarotero/psr7-middlewares](https://github.com/oscarotero/psr7-middlewares/blob/v3.10.2/src/Middleware/FormatNegotiator.php) Unfortunately this lib isnt *Strategility Compatible*